.codeclimate.yml

```diff
@@ -0,0 +1,55 @@
+---
+exclude_paths:
+- ".git/"
+- "**.gif"
+- "**.html"
+- "**.json"
+- "**.png"
+- "**.svg"
+- "**.xml"
+- "**.yaml"
+- "**.yml"
+- "locale/"
+- "spec/"
+- "tools/"
```

Comment:
I'd only exclude things you actually have in this repo.  locale?  tools?  svg?  xml?, etc.